src/eventDispatchers/mouseEventHandlers/mouseDown.js

```diff
@@ -48,14 +49,12 @@ export default function (evt) {
   if (activeTools.length > 0) {
     // TODO: If length > 1, you could assess fitness and select the ideal tool
     // TODO: But because we're locking this to 'active' tools, that should rarely be an issue
+    // Super-Meta-TODO: ^ I think we should just take the approach of one active tool per mouse button?
```

Comment:
I mean... Probably, haha. It's still a little early to put in the constraint, but I imagine that's what will shake out. Having more than one active tool per mouse button, touch event type, keybinding, etc. feels messy.

Reply:
;). Time will tell.